From: Miaohe Lin <linmiaohe@huawei.com>
Subject: MAINTAINERS: add maintainer information for z3fold
Date: Mon, 30 May 2022 19:45:05 +0800

Add the maintainer information for the z3fold compressed page allocator. 
Also Add myself as a reviewer.

Link: https://lkml.kernel.org/r/20220530114505.52367-1-linmiaohe@huawei.com
Signed-off-by: Miaohe Lin <linmiaohe@huawei.com>
Cc: Vitaly Wool <vitaly.wool@konsulko.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 MAINTAINERS |    7 +++++++
 1 file changed, 7 insertions(+)

--- a/MAINTAINERS~maintainers-add-maintainer-information-for-z3fold
+++ a/MAINTAINERS
@@ -21928,6 +21928,13 @@ L:	linux-mm@kvack.org
 S:	Maintained
 F:	mm/zbud.c
 
+Z3FOLD COMPRESSED PAGE ALLOCATOR
+M:	Vitaly Wool <vitaly.wool@konsulko.com>
+R:	Miaohe Lin <linmiaohe@huawei.com>
+L:	linux-mm@kvack.org
+S:	Maintained
+F:	mm/z3fold.c
+
 ZD1211RW WIRELESS DRIVER
 M:	Ulrich Kunitz <kune@deine-taler.de>
 L:	linux-wireless@vger.kernel.org
_
